packages/gatsby-transformer-screenshot/src/placeholder-file-node.js

```diff
@@ -0,0 +1,19 @@
+const path = require(`path`)
+const { createFileNode } = require(`gatsby-source-filesystem/create-file-node`)
```

Comment:
this could be cleaner if we would export that from `gatsby-source-filesystem` - probably it would need other name as it doesn't really create nodes - just prepare node object